src/amo/store.js

```diff
@@ -19,27 +19,29 @@ import search from 'core/reducers/search';
 import { middleware } from 'core/store';
 
 
+export const AMO_REDUCERS = {
```

Comment:
any reason why this is in all caps? I feel like that's a bad habit we've gotten into.